src/components/tabs/Tabs.tsx

```diff
@@ -1,8 +1,7 @@
-import React, { ReactComponentElement, ReactNode, useEffect, useState } from "react";
-import { STabs } from "./styles/STabs";
-import { STabsItemBorder } from "./styles/STabsItemBorder";
-import { STabsItem } from "./styles/STabsItem";
+import React, { ReactNode, useEffect, useState } from "react";
+import { LineTabs } from "..";
```

Comment:
while this works, it is safer to make sure we reference the proper module by providing the name of the file.